identityClient/identityClient.go

```diff
@@ -16,10 +19,61 @@ type E3dbIdentityClient struct {
 	authClient *http.Client
 }
 
+// ListRealms lists the realms belonging to the requester returning the realms and error (if any).
```

Comment:
does the endpoint provide authZ based on identifying information in the signature?

Reply:
yes, its protected by cyclops to enforce only queen client tsv1 authenticated requests arrive at this endpoint.